commit 213bb73d5ddfbfbbe98b52762751ed155df05c9d
Author:     Tizen Build Automation <tizenbuild@samsung.com>
AuthorDate: Sun Oct 25 01:41:08 2026 +0000
Commit:     Tizen Build Automation <tizenbuild@samsung.com>
CommitDate: Sun Oct 25 01:41:08 2026 +0000

    Update candidate_snapshot : TIZEN/Tizen/Tizen-Base-Toolchain/project_meta,TIZEN/Tizen/Tizen-Unified-Toolchain/project_meta,TIZEN/Tizen/Tizen-Unified-X/project_meta,TIZEN/Tizen/Tizen-Unified/project_meta,TIZEN/Tizen/Tizen-Base/project_meta,TIZEN/Tizen-10.1/Tizen-10.1-Unified/project_meta,TIZEN/Tizen-10.0/Tizen-10.0-Unified/project_meta,
---
 TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot | 1 -
 TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot | 2 --
 TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot    | 2 --
 TIZEN/Tizen/Tizen-Base/candidate_snapshot              | 1 -
 TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot | 1 -
 TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot         | 2 --
 TIZEN/Tizen/Tizen-Unified/candidate_snapshot           | 1 -
 7 files changed, 10 deletions(-)

diff --git a/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot b/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
index 0941f98..cad7bd8 100644
--- a/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
+++ b/TIZEN/Tizen-10.0/Tizen-10.0-Unified/candidate_snapshot
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
-  <candidate target="tizen-10.0-unified_20261014.065551" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261015.031059" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261015.031153" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
   <candidate target="tizen-10.0-unified_20261015.092703" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.0/Tizen-10.0-Unified/" />
diff --git a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
index aeb9d8b..ad00608 100644
--- a/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
+++ b/TIZEN/Tizen-10.1/Tizen-10.1-Unified/candidate_snapshot
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
-  <candidate target="tizen-10.1-unified_20261014.010544" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
-  <candidate target="tizen-10.1-unified_20261014.210343" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261015.031743" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261015.092932" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
   <candidate target="tizen-10.1-unified_20261016.022138" url="http://download.tizen.org/snapshots/TIZEN/Tizen-10.1/Tizen-10.1-Unified/" />
diff --git a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
index 70d9c23..ec9fee6 100644
--- a/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base-Toolchain/candidate_snapshot
@@ -4,14 +4,12 @@
   <candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <candidate target="tizen-base-toolchain_20261014.065008" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261014.065008" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Base/candidate_snapshot b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
index c0a7500..552cf92 100644
--- a/TIZEN/Tizen/Tizen-Base/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Base/candidate_snapshot
@@ -11,7 +11,6 @@
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261014.065008" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
index 629fc1a..1fd70a3 100644
--- a/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-Toolchain/candidate_snapshot
@@ -8,7 +8,6 @@
   <base_candidate target="tizen-base-toolchain_20260924.094908" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.024319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261013.140248" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
-  <base_candidate target="tizen-base-toolchain_20261014.065008" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261015.022255" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
   <base_candidate target="tizen-base-toolchain_20261020.074527" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Base-Toolchain/" />
 </snapshotinfo>
\ No newline at end of file
diff --git a/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
index 599be4e..505120d 100644
--- a/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified-X/candidate_snapshot
@@ -1,8 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
-  <candidate target="tizen-unified-x_20261014.022319" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
-  <candidate target="tizen-unified-x_20261014.230217" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <candidate target="tizen-unified-x_20261015.092750" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <candidate target="tizen-unified-x_20261015.211107" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
   <candidate target="tizen-unified-x_20261018.221821" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified-X/" />
diff --git a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
index 6f6ef30..3f83615 100644
--- a/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
+++ b/TIZEN/Tizen/Tizen-Unified/candidate_snapshot
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <snapshotinfo>
   <candidate target="reference" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
-  <candidate target="tizen-unified_20261014.063636" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261015.005108" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261015.101045" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
   <candidate target="tizen-unified_20261015.232434" url="http://download.tizen.org/snapshots/TIZEN/Tizen/Tizen-Unified/" />
